analytics/terraform/datahub-on-eks/datahub-eks/values/datahub_values.yaml

```diff
@@ -0,0 +1,334 @@
+# Values to start up datahub after starting up the datahub-prerequisites chart with "prerequisites" release name
+# Copy this chart and change configuration as needed.
+datahub-gms:
+  enabled: true
+  image:
+    repository: linkedin/datahub-gms
+    tag: "v0.8.45"
```

Comment:
Users should be able to just increment the Helm version so that they can get the latest image related to the version. Use only modified values and ignore all the defaults such as `tag: "v0.8.45"` etc.

Reply:
it was copied from an older version of the datahub helm example.  removed